test/ovn/ovn.dl

```diff
@@ -1,2 +1,17 @@
 import OVN_Northbound
 import OVN_Southbound
+
+
+// FlatMap test
+input relation HostAddresses(host: bit<64>, addrs: Set<string>)
+relation HostAddress(host: bit<64>, addr: string)
+
+HostAddress(host, addr) :- HostAddresses(host, addrs),
+                           FlatMap(addr=addrs).
```

Comment:
I also find this syntax a little jarring. I would prefer something like FlatMap(addr in addrs).

Reply:
makes sense.